stockAndPrices/services/app/main.py

```diff
@@ -1,7 +1,12 @@
 from flask import Flask, request
+from flask_cors import CORS
+from configuration.Origin import Origin
 
 
 app = Flask(__name__)
+app.config.from_pyfile("configuration/DevConfig.py")
+
+cors = CORS(app, resources={r"/.*": {"origins": Origin}})
```

Comment:
are we allowing all origins to hit, or just the react app origin here?

Reply:
Yes, "Origin" was imported from Origin.py in the configuration folder,